### PR TITLE
Documentation review

### PR DESCRIPTION
**Pull request by whitx**
For review and merge

### CODE REVIEW

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 1]
```diff
@@ -2171,12 +2172,10 @@ To assign a role by tiers (example: slow, medium and fast), edit the file `lib/p
 
 *destination_url* is the url that the device will be redirected after a successful authentication.
 
-CAUTION: The use of different billing tiers requires different roles in PacketFence. Make sure to create these roles first otherwise you will run into problems.
-
 Portal Profiles
 ~~~~~~~~~~~~~~~
 
-In some cases, you may want to present a different captive portal (see below for the available customizations) according to the SSID, the VLAN, the switch IP/MAC or the URI the client connects to. To do so, PacketFence has the concept of portal profiles which gives you this possibility. 
+In some cases, you may want to present a different captive portal (see below for the available customizations) according to the SSID, the VLAN, the switch IP/MAC or the URL the client connects to. To do so, PacketFence has the concept of portal profiles which gives you this possibility.
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: Keep URI

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -2209,7 +2208,7 @@ You must also enable the passthrough option in your PacketFence configuration (t
 
 Google
 ^^^^^^
-In order to use Google as a OAuth2 provider, you need to get an API key to access their services.  Sign up here : http://code.google.com/apis/console.  Make sure you use this URI for the "Redirect URI"
+In order to use Google as a OAuth2 provider, you need to get an API key to access their services.  Sign up here : http://code.google.com/apis/console.  Make sure you use this URL for the "Redirect URL"
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: Keep URI

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 3]
```diff
@@ -3300,7 +3299,7 @@ The last block defines the relationship between the tests and the desired action
   action = 501
   redirect_url =
 
-This filter will return an error code (501) if the user agent is Dalvik and the URI doesn't contain _/generate_204.
+This filter will return an error code (501) if the user agent is Dalvik and the URL doesn't contain _/generate_204.
```
Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc]: Keep URI

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 4]
```diff
@@ -2139,6 +2138,8 @@ Billing engine parameters are specified in `conf/pf.conf` or from *Configuration
 It is also possible to configure multiple network access with different prices.
 For example, you may want to provide basic Internet access with a decent speed at a specific price and another package with high speed connection at another price.
 
+CAUTION: The use of different billing tiers requires different roles in PacketFence. Make sure to create these roles first otherwise you will run into problems.
```
Comment (extrafu) [inline docs/PacketFence_Administration_Guide.asciidoc:2141]: There must be a directive for "cautions" in asciidoc, something that gets formatted correctly afterwards.

Comment (cgx) [inline docs/PacketFence_Administration_Guide.asciidoc:2141]: Yes! So this prefix is correct. FYI -- we can also use `NOTE:`, `TIP:`, `IMPORTANT:`, and `WARNING:`.

**Pull request merged**